### PR TITLE
Add missing reg type to MCAPS 

### PR DESCRIPTION
**Pull request by Alxlmsdn**

### CODE REVIEW

**Comment by Alxlmsdn** [comment]
@nelson-w

**Comment by nelson-w** [comment]
@Alxlmsdn to confirm whether the whitespace is intentional from EC

**Comment by Alxlmsdn** [comment]
@nelson-w have corrected attendee type

File: resource/mcaps2022-prod/rules.json [hunk 1]
```diff
@@ -2,7 +2,9 @@
   "UserAccess": [
     {
       "Identifiers": [
-        "Anonymous"
+        "Anonymous",
+        "Speaker External Digital",
```
Comment (nelson-w) [inline resource/mcaps2022-prod/rules.json:6]: remove

File: resource/mcaps2022-test/rules.json [hunk 2]
```diff
@@ -2,7 +2,9 @@
   "UserAccess": [
     {
       "Identifiers": [
-        "Anonymous"
+        "Anonymous",
+        "Speaker External Digital",
```
Comment (nelson-w) [inline resource/mcaps2022-test/rules.json:6]: remove, just have anonymous here

**Review by nelson-w — APPROVED** [review]

**Pull request merged**